README.md

```diff
@@ -25,6 +25,52 @@ juju deploy temporal-worker-k8s
 juju config temporal-worker-k8s --file=path/to/config.yaml
 ```
 
+### Attaching "workflows-file" resource
+
+The Temporal worker operator expects a "workflows-file" resource to be attached
+after deployment, which contains a set of defined Temporal workflows and
+activities as defined in the [resource_sample](./resource_sample/) directory.
+The structure of the built wheel file must follow the same structure:
+
+```
+- workflows/
+    - workflow-a.py
+    - workflow-b.py
+- activities/
+    - activity-a.py
+    - activity-b.py
+- some_other_direcory/
+- some_helper_file.py
```

Comment:
Does `some_helper_file.py` need to be in `some_other_directory`?

Reply:
Not necessarily, just a way of showing that the user can have any folder structure they want as long as they have the mandatory ```workflows``` and ```activities``` directories.